### PR TITLE
chore(docs): improve Bootstrap/BootstrapVue style import guide in "Getting started" docs

### PR DESCRIPTION
**Pull request by darrelfrancis**
The original version was a little confusing because it was repetitive and missed the opportunity to make clear at the outset that there are potentially 3 SCSS files and they must be imported in order.

### Describe the PR

Simplify the documentation making clear the importance of the order, and exactly how to import

### PR checklist

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix (fixes a boo-boo in the code) - `fix(...)`, requires a patch version update
- [ ] Feature (adds a new feature to BootstrapVue) - `feat(...)`, requires a minor version update
- [ ] Enhancement (augments an existing feature) - `feat(...)`, requires a minor version update
- [ ] ARIA accessibility (fixes or improves ARIA accessibility) - `fix(...)`, requires a patch or minor version update
- [x] Documentation update (improves documentation or typo fixes) - `chore(docs)`, requires a patch version update
- [ ] Other (please describe)

**Does this PR introduce a breaking change?** (check one)

- [x] No
- [ ] Yes (please describe since breaking changes require a minor version update)

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [    ] When resolving a specific issue, it's referenced in the PR's title (i.e. `[...] (fixes #xxx[,#xxx])`, where "xxx" is the issue number)
- [x] It should address only one issue or feature. If adding multiple features or fixing a bug and adding a new feature, break them into separate PRs if at all possible.
- [x] The title should follow the [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e. `fix(alert): not alerting during SSR render`, `docs(badge): update pill examples`, `chore(docs): fix typo in README`, etc.). **This is very important, as the `CHANGELOG` is generated from these messages, and determines the next version type (patch or minor).**

### CODE REVIEW

File: docs/markdown/intro/README.md [hunk 1]
```diff
@@ -126,45 +124,50 @@ npm install vue bootstrap-vue bootstrap
...
 
-// Install BootstrapVue
+// Tell Vue to make BootstrapVue available throughout your project
 Vue.use(BootstrapVue)
 // Optionally install the BootstrapVue icon components plugin
 Vue.use(IconsPlugin)
 ```
 
-And import Bootstrap and BootstrapVue `css` files:
+You must also import the `css` files. This consists of 2 or 3 files in this specific order: your own custom `scss` file (for example if you are globally redefining the 'primary' color) and the Bootstrap and BootstrapVue `css` files. There are two ways of doing this.
```
Comment (Hiws) [inline docs/markdown/intro/README.md:140]: > (for example if you are globally redefining the 'primary' color)
I'm not sure I understand this section.
If you're importing `Bootstrap.css`, then it wont matter what changes you've made in your own `scss` file, as it wont compile bootstrap with it. Or am I misunderstanding it?

Comment (jackmu95) [reply docs/markdown/intro/README.md]: When importing the Bootstrap *CSS* `dist` files, there is no way to change/redefine variables.
This is only possible when using the SCSS files.

Comment (jackmu95) [reply docs/markdown/intro/README.md]: The important part is already quite well described:

> Be sure to @import or define your custom variable values before including Bootstrap SCSS (bootstrap.scss), and include BootstrapVue SCSS (bootstrap-vue.scss) after that to ensure variables are set up correctly.

Comment (darrelfrancis) [reply docs/markdown/intro/README.md]: Ah I missed that subtlety, sorry! I was careless between css and scss. I will correct and come back to you.

**Review by jackmu95 — APPROVED** [review]

**Comment by jackmu95** [comment]
@darrelfrancis I made some final adjustments. Are you fine with that?

**Review by Hiws — APPROVED** [review]

**Review by darrelfrancis — COMMENTED** [review]
Looks great to me, thank you!

**Pull request merged**